tests/driver_srf04/Makefile

```diff
@@ -0,0 +1,13 @@
+APPLICATION = driver_srf04
+
+BOARD ?= calliope-mini
+
+RIOTBASE ?= $(CURDIR)/../..
+
+FEATURES_REQUIRED = periph_gpio
+
+USEMODULE += srf04
+USEMODULE += xtimer
```

Comment:
Since xtimer is used internally by the driver, it should be added as a dependency of the driver module in drivers/Makefile.dep